cime_config/tests.py

```diff
@@ -299,7 +299,10 @@
             "SMS_D_Ld1.T62_oQU240.GMPAS-IAF.mpaso-freshwater_tracers",
             "ERS_Ld5_D.T62_oQU240.GMPAS-IAF.mpaso-conservation_check",
             "ERS_Ld5_PS.ne30pg2_r05_IcoswISC30E3r5.CRYO1850-DISMF.mpaso-scaled_dib_dismf",
-            "ERS_Ld5.TL319_oQU240wLI_gis20.MPAS_LISIO_JRA1p5.mpaso-ocn_glc_tf_coupling",
+            # OCN/GLC 3d TF coupling GIS test:
```

Comment:
should we run this e3sm_ocnice_stealth_features suite somewhere on a regular basis?

Reply:
@rljacob -- it runs every night on gcp12

Reply:
Oh I see.  Its included in extra coverage.